{{cookiecutter.project_name}}/tests/test_token.py

```diff
@@ -245,7 +246,7 @@ def test_permit(chain,token,owner):
         token.permit(owner, spender, amount+1, deadline, signature, sender=spender)
     with ape.reverts():
         token.permit(owner, spender, amount, deadline+1, signature, sender=spender)
-    
+
     token.permit(owner, spender, amount, deadline, signature, sender=spender)
```

Comment:
So this `permit()` calls still fails on my end, but with an empty message in the `TransactionFailed` exception:

<img width="1421" alt="image" src="https://user-images.githubusercontent.com/108302637/176315168-5fe53b6a-b242-446a-9a2d-7d611f7785ff.png">

Reply:
Evan is thinking there's maybe something up with the encoded signature, but yeah no error message makes it hard to know

Reply:
Someone at ETH-NYC reported an issue with `sign_message()`. I wonder if its related?